ash/src/vk.rs

```diff
@@ -49,4 +49,17 @@ pub trait Handle {
     const TYPE: ObjectType;
     fn as_raw(self) -> u64;
     fn from_raw(_: u64) -> Self;
+
+    /// Returns true if the handle is a null handle.
```

Comment:
```suggestion
    /// Returns [`true`] if the handle is a null handle.
```

Or maybe just "Returns whether the handle is ..."